packages/fxa-geodb/lib/fxa-geodb.js

```diff
@@ -6,14 +6,17 @@ var DEFAULTS = require('./defaults');
 var ERRORS = require('./errors');
 var maxmind = require('maxmind');
 var Location = require('./location');
+const fs = require('fs');
 
 module.exports = function (options) {
   'use strict';
 
   options = options || {};
   var dbPath = options.dbPath || DEFAULTS.DB_PATH;
 
-  const dbLookup = maxmind.openSync(dbPath);
```

Comment:
Guess this got deprecated but don't see a reason why? They have examples of loading the db using `fs` so went with that.

Reply:
Looks like they might have dropped support for `.openSync` in 3.x when they promisified the API:
https://github.com/runk/node-maxmind/commit/b0a93ab74ee9c7e884711262b8cb312a6d6bbd2a